doc/source/theory/multiphysics/heat_transfer/heat_equation.rst

```diff
@@ -2,4 +2,50 @@
 Heat Transfer Equations
 ================================
 
+In Lethe, it is possible to solve the heat equation in various loading conditions. The main equation is derived from the energy equation in incompressible fluids. Assuming a constant heat capacity :math:`c_p` and dynamic viscosity :math:`\mu`, the equation takes the following form: 
+
+.. math::
+    \rho c_p \frac{\partial T}{\partial t} + \rho c_p (\mathbf{u} \cdot \nabla)T - k \nabla^2 T = - \phi + Q
```

Comment:
I agree for the theoretical section of the doc. For the FEM section, this would add a term which, from a quick look into the HT assembler, doesn't seem to be present in Lethe's formulation. What do you think of keeping it as is in the FEM section and specifying that the weak form shows k as a constant?

Reply:
I am not sure that there is a new term appearing, but maybe I am to tired to focus. We can ask Pierre for its opinion.